src/Schema/MySQLSchemaManager.php

```diff
@@ -192,6 +195,10 @@ protected function _getPortableTableColumnDefinition(array $tableColumn): Column
             case 'year':
                 $length = null;
                 break;
+
+            case 'enum':
+                $values = explode('\',\'', substr($tableColumn['type'], 6, -2));
```

Comment:
can MySQL enum values contain single quotes (through some kind of escaping) ?

Reply:
Probably. I'll check that.

Reply:
Good catch. I think I have it covered now.